### ISSUE TITLE
Inline loading of commands would be cleaner

### ISSUE THREAD

**Issue by scepticalcat** [KarthickNarayana/toy_robot#2]
https://github.com/KarthickNarayana/toy_robot/blob/3d570a92d5c8b027e2456ec4f7bce47223719ecf/exe/toy_robot.rb#L6

This feels akward for a number of reasons, specifically having to assign the return value of `load_commands` to a variable that is passed back in with the following call to `run`. Consider that we're working with an instance, so we could either pass the commands file as an argument to the `initialize` method removing the need for a public `load_commands` call, or, still call `load_commands` but store the result as an instance var internally so in either case we then can call `run` without additional arguments.

**Comment by KarthickNarayana** [comment]
Agreed.  In the main program - or the program which is the entry point for the input command. I can always create a new Robot Object with a boundary set to the Robot and then take the commands as another parameter and execute the commands on that instance of a Robot.  

This way the Robot knows about the boundary which is in 2D - and if you need the Robot to climb instead of just move we can implement the new boundary independently.
